### PR TITLE
first commits

### PR DESCRIPTION
**Pull request by itsmostafa**
Initialize this repo by pulling the latest binary from [con-pca-tasks](github.com/cisagov/con-pca-tasks) repo

## 🗣 Description ##
Update dockerfile & docker compose to pull and run the latest binary from con-pca-tasks
Initialize other files to reflect the scope of this project

## 🧪 Testing ##
testing by building and running the docker image locally

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ __pycache__
 .mypy_cache
 .pytest_cache
 .python-version
+.venv
```
Comment (jsf9k) [inline .gitignore:13]: Are you not using `pyenv`?

File: Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ ARG VERSION
 # suggested that you use an email address here that is specific to the
 # person or group that maintains this repository; for example:
 # LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
-LABEL org.opencontainers.image.authors="github@cisa.dhs.gov"
```
Comment (jsf9k) [inline Dockerfile:19]: Now that you've updated this, you can remove the comment just about the email address.  Be careful not to remove the comment about the annotation keys.

File: Dockerfile [hunk 3]
```diff
@@ -52,41 +52,18 @@ RUN addgroup --system --gid ${CISA_GID} ${CISA_GROUP} \
...
-# slightly longer install times.
-###
-RUN wget --output-document sourcecode.tgz \
-    https://github.com/cisagov/skeleton-python-library/archive/v${VERSION}.tar.gz \
-    && tar --extract --gzip --file sourcecode.tgz --strip-components=1 \
-    && pip install --no-cache-dir --requirement requirements.txt \
-    && ln -snf /run/secrets/quote.txt src/example/data/secret.txt \
-    && rm sourcecode.tgz
+# Install Con-PCA-Tasks binary file
+RUN wget "https://github.com/cisagov/con-pca-tasks/releases/download/v0.0.1/pca-linux-amd64"
```
Comment (jsf9k) [inline Dockerfile]: These quotes are unnecessary.  Please remove them.

File: README.md [hunk 4]
```diff
@@ -161,34 +77,25 @@ The following ports are exposed by this container:
...
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| 8080 | Con-PCA Tasks  |
 
 ## Environment variables ##
 
 ### Required ###
 
-There are no required environment variables.
+There are no required environment variables as of now.
```
Comment (jsf9k) [inline README.md]: I suggest just leaving this as it in the skeleton, since your change doesn't really add anything.

File: README.md [hunk 5]
```diff
@@ -1,103 +1,35 @@
...
 [![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/con-pca-tasks-docker/tags)
 
-This is a Docker skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub Docker project
-started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit hooks](https://pre-commit.com)
-and [GitHub Actions](https://github.com/features/actions) configurations
-appropriate for Docker containers and the major languages that we use.
+This project's purpose is to containerize [Con-PCA-Tasks](https://github.com/cisagov/con-pca-tasks)
+project as part of our cicd pipeline.
```
Comment (jsf9k) [inline README.md]:
```suggestion
project as part of our CI/CD pipeline.
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested some minor changes, but great work!

File: .gitignore (hunk 1 above)
Comment (itsmostafa) [reply .gitignore:13]: @jsf9k no i'm using virtualenv using `python -m venv` I make sure to create / recreate the virtual environment using the latest stable version of python when needed.

File: Dockerfile (hunk 2 above)
Comment (itsmostafa) [reply Dockerfile:19]: @jsf9k resolved with [0327b1d](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/0327b1d4efe6104d270b9164d1bdce93aee85d32)

File: Dockerfile (hunk 3 above)
Comment (itsmostafa) [reply Dockerfile]: @jsf9k resolved with [0327b1d](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/0327b1d4efe6104d270b9164d1bdce93aee85d32)

File: README.md (hunk 4 above)
Comment (itsmostafa) [reply README.md]: @jsf9k resolved with [6a7524e](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/6a7524ea2170f8ceb79a123a7de4f04873f1b822)

File: README.md (hunk 5 above)
Comment (itsmostafa) [reply README.md]: @jsf9k resolved with [6a7524e](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/6a7524ea2170f8ceb79a123a7de4f04873f1b822)

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -27,10 +27,10 @@ on:
 env:
   BUILDX_CACHE_DIR: ~/.cache/buildx
   CURL_CACHE_DIR: ~/.cache/curl
-  IMAGE_NAME: cisagov/example
+  IMAGE_NAME: cisagov/con-pca-tasks
   PIP_CACHE_DIR: ~/.cache/pip
   PLATFORMS: "linux/amd64,linux/arm/v6,linux/arm/v7,\
-  linux/arm64,linux/ppc64le,linux/s390x"
+    linux/arm64,linux/ppc64le,linux/s390x"
```
Comment (dav3r) [inline .github/workflows/build.yml]: You should revert any unnecessary changes from the skeleton:
```suggestion
  linux/arm64,linux/ppc64le,linux/s390x"
```

File: README.md [hunk 7]
```diff
@@ -1,103 +1,35 @@
...
-## Running ##
-
-### Running with Docker ###
+## Related Con-PCA Repositories ##
 
-To run the `cisagov/example` image via Docker:
+- [con-pca-api](https://github.com/cisagov/con-pca-api)
+- [con-pca-cicd](https://github.com/cisagov/con-pca-cicd)
+- [con-pca-web](https://github.com/cisagov/con-pca-web)
+- [con-pca-tasks](https://github.com/cisagov/con-pca-web)
```
Comment (dav3r) [inline README.md]: Copy/paste error:
```suggestion
- [con-pca-tasks](https://github.com/cisagov/con-pca-tasks)
```

File: README.md [hunk 8]
```diff
@@ -114,40 +46,24 @@ environment variables.  See the
...
 |-----------|-------------|
-|`cisagov/example:1.2.3`| An exact release version. |
-|`cisagov/example:1.2`| The most recent release matching the major and minor version numbers. |
-|`cisagov/example:1`| The most recent release matching the major version number. |
-|`cisagov/example:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
-|`cisagov/example:nightly` | A nightly build of the `develop` branch of this repository. |
-|`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
+|`cisagov/con-pca-tasks:1.2.3`| An exact release version. |
+|`cisagov/con-pca-tasks:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/con-pca-tasks:1`| The most recent release matching the major version number. |
```
Comment (dav3r) [inline README.md]: Our standard is to have the versions in this documentation align with the version of the Docker image currently being used by this repo (`v0.0.1`):
```suggestion
|`cisagov/con-pca-tasks:0.0.1`| An exact release version. |
|`cisagov/con-pca-tasks:0.0`| The most recent release matching the major and minor version numbers. |
|`cisagov/con-pca-tasks:0`| The most recent release matching the major version number. |
```

File: README.md [hunk 9]
```diff
@@ -161,10 +77,7 @@ The following ports are exposed by this container:
 
 | Port | Purpose        |
 |------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| 8080 | Con-PCA Tasks  |
```
Comment (dav3r) [inline README.md]: Can the purpose of port 8080 be explained a bit more here?

File: README.md [hunk 10]
```diff
@@ -178,17 +91,11 @@ There are no required environment variables.
...
-| Name  | Purpose | Default |
-|-------|---------|---------|
-| `ECHO_MESSAGE` | Sets the message echoed by this container.  | `Hello World from Dockerfile` |
-
 ## Secrets ##
 
 | Filename     | Purpose |
 |--------------|---------|
-| `quote.txt` | Replaces the secret stored in the example library's package data. |
+| `quote.txt` | Replaces the secret stored in the library's package data. |
```
Comment (dav3r) [inline README.md]: If you are not using the example secret, it should be deleted and this section of the README should be updated to reflect that.

File: docker-compose.yml [hunk 11]
```diff
@@ -29,9 +27,9 @@ services:
       - source: quote_txt
         target: quote.txt
```
Comment (dav3r) [inline docker-compose.yml]: As mentioned elsewhere, the `secrets` block can be removed if you are not using it.

File: Dockerfile [hunk 12]
```diff
@@ -52,41 +46,18 @@ RUN addgroup --system --gid ${CISA_GID} ${CISA_GROUP} \
...
-###
-RUN wget --output-document sourcecode.tgz \
-    https://github.com/cisagov/skeleton-python-library/archive/v${VERSION}.tar.gz \
-    && tar --extract --gzip --file sourcecode.tgz --strip-components=1 \
-    && pip install --no-cache-dir --requirement requirements.txt \
-    && ln -snf /run/secrets/quote.txt src/example/data/secret.txt \
-    && rm sourcecode.tgz
+# Install Con-PCA-Tasks binary file
+RUN wget https://github.com/cisagov/con-pca-tasks/releases/download/v0.0.1/pca-linux-amd64
+RUN mv pca-linux-amd64 /bin/pca
```
Comment (dav3r) [inline Dockerfile]: Why are you only using the amd64 release here?  What about the other platforms?

**Review by dav3r — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (itsmostafa) [reply README.md]: @dav3r port 8080 is the common port number used for web apps built in golang (per documentation and tutorials i've followed in the past). No other reason besides that. Do you suggest I change it to something else?

Comment (jsf9k) [reply README.md]: @dav3r is just asking you to expand on the description (currently "Con-PCA Tasks"). For those of us who are unfamiliar with Con-PCA, this description does not convey any information.  What is served on port 8080?  A REST API that allows one to add/remove/modify PCA campaigns?  Something else?

File: .gitignore (hunk 1 above)
Comment (jsf9k) [reply .gitignore:13]: I'll resolve this, but I highly suggest using `pyenv` over the `venv` module.  It's a game changer.

File: README.md (hunk 9 above)
Comment (itsmostafa) [reply README.md]: Ah okay, Understood. Working on that now. Sorry about that @dav3r

File: README.md (hunk 7 above)
Comment (itsmostafa) [reply README.md]: @dav3r resolved with [51f2f74](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/51f2f74eae3848774a376a6aca16ec581938b0cc)

File: README.md (hunk 8 above)
Comment (itsmostafa) [reply README.md]: @dav3r resolved with [51f2f74](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/51f2f74eae3848774a376a6aca16ec581938b0cc)

File: README.md (hunk 10 above)
Comment (itsmostafa) [reply README.md]: @dav3r resolved with [51f2f74](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/51f2f74eae3848774a376a6aca16ec581938b0cc)

File: docker-compose.yml (hunk 11 above)
Comment (itsmostafa) [reply docker-compose.yml]: @dav3r resolved with [598d748](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/598d748b6771cf70e8fde93a7114349e7fab7e4f)

File: Dockerfile (hunk 12 above)
Comment (itsmostafa) [reply Dockerfile:57]: @dav3r we currently use a `db.r5.xlarge` instance which is running on x86-64 architecture to run our other containers in Con-PCA. We're probably going to spin up a similar instance for this project as well.

Also I'm assuming its more common that users pulling this image will build a container on an x86-64 machine. And those who do have arm64 will be able to run it anyways due to emulation. 

I don't think we need other platforms for this image build but if one would like to run an executable directly, i have other platforms supported with each release here: https://github.com/cisagov/con-pca-tasks/releases/tag/v0.0.1

Please let me know your thoughts or how you'd like to me to go forward. Thanks!

File: README.md (hunk 9 above)
Comment (itsmostafa) [reply README.md]: @dav3r resolved with [51f2f74](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/51f2f74eae3848774a376a6aca16ec581938b0cc)

File: .github/workflows/build.yml (hunk 6 above)
Comment (itsmostafa) [reply .github/workflows/build.yml]: @dav3r resolved with [b899561](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/b899561c88411ce3f4fdfc15f6e3266bb2a98bda) However, I get this error showing when I revert it back from this change

Screen Shot 2023-02-09 at 10 11 58 AM

File: Dockerfile (hunk 12 above)
Comment (itsmostafa) [reply Dockerfile]: @dav3r resolved with [8221b09](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/8221b092804fd499d96dd73d404f8ebe49851494)

I implemented Nick's solution to choose the correct build based on the target's arch type ^
thanks @mcdonnnj

File: .github/workflows/build.yml [hunk 13]
```diff
@@ -262,27 +261,27 @@ jobs:
...
-              fromJson(needs.prepare.outputs.repometa).name }}
+            fromJson(needs.prepare.outputs.repometa).name }}
 
             org.opencontainers.image.url=${{
-              fromJson(needs.prepare.outputs.repometa).html_url }}
+            fromJson(needs.prepare.outputs.repometa).html_url }}
 
             org.opencontainers.image.version=${{
-              needs.prepare.outputs.source_version }}"
+            needs.prepare.outputs.source_version }}"
```
Comment (jsf9k) [inline .github/workflows/build.yml]: Do not make unnecessary changes to the skeleton code.  This path leads to [The Tree of Woe](https://conan.fandom.com/wiki/Tree_of_Woe).

File: .github/workflows/build.yml [hunk 14]
```diff
@@ -391,27 +390,27 @@ jobs:
...
```
Comment (jsf9k) [inline .github/workflows/build.yml]: Do not make unnecessary changes to the skeleton code.  This path leads to [The Tree of Woe](https://conan.fandom.com/wiki/Tree_of_Woe).

File: README.md [hunk 15]
```diff
@@ -159,12 +75,9 @@ Hub for a list of all the supported tags.
...
 
-| Port | Purpose        |
-|------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| Port | Purpose                                             |
+|------|-----------------------------------------------------|
+| 8080 | Rest API to manage tasks and trigger sending emails |
```
Comment (jsf9k) [inline README.md]: Very minor change to improve readability - take it or leave it as you see fit:
```suggestion
| 8080 | REST API to manage tasks and trigger the sending of emails |
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
I have a trio of suggestions:
- Two to remove unnecessary changes to skeleton code (this will save you a lot of heartache in the future)
- One language suggestion that you can take or leave as you see fit

File: .github/workflows/build.yml (hunk 6 above)
Comment (dav3r) [reply .github/workflows/build.yml]: Are you only seeing that warning in your editor?  As long as there are no pre-commit or GitHub Actions failures, there's no issue from our perspective.

However, if you feel you have a good justification for doing so, you could create a small PR to change [those lines in skeleton-docker](https://github.com/cisagov/skeleton-docker/blob/83baee5da2c5dde558cda41a5cccfefec7c2586c/.github/workflows/build.yml#L32-L33).

File: README.md (hunk 8 above)
Comment (dav3r) [reply README.md]: @itsmostafa Reminder that you can use the GitHub web UI to commit suggestions.  In addition to supporting the best practice of keeping commits small and targeted, it also automatically gives co-author commit credit to whoever makes the suggestion.  If you do use this feature, please be sure to replace GitHub's default commit message (the not-very-helpful "Update [filename]") with something more meaningful.

File: README.md (hunk 9 above)
Comment (dav3r) [reply README.md]: Thanks @itsmostafa - much better! 👍

File: .github/workflows/build.yml [hunk 16]
```diff
@@ -29,8 +29,7 @@ env:
   CURL_CACHE_DIR: ~/.cache/curl
   IMAGE_NAME: cisagov/con-pca-tasks
   PIP_CACHE_DIR: ~/.cache/pip
-  PLATFORMS: "linux/amd64,linux/arm/v6,linux/arm/v7,\
-  linux/arm64,linux/ppc64le,linux/s390x"
+  PLATFORMS: "linux/amd64,linux/arm64"
```
Comment (dav3r) [inline .github/workflows/build.yml:33]: If we ever decide to remove platforms, it's best to include a comment with some explanation of why we are doing so.  Can you please add a brief comment here?

File: Dockerfile (hunk 12 above)
Comment (dav3r) [reply Dockerfile]: Looks like this change was done in https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/efa61943bef119a83359dbf421dd50fc846e52a2, not https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/8221b092804fd499d96dd73d404f8ebe49851494.

File: README.md (hunk 8 above)
Comment (dav3r) [reply README.md]: One other benefit of using the "Commit suggestion" feature is that GitHub automatically resolves the conversation, so it saves a little bit of time for the reviewer.

File: .github/workflows/build.yml (hunk 6 above)
Comment (itsmostafa) [reply .github/workflows/build.yml]: @dav3r i'm okay with leaving it alone. I believe this is just coming from my editor. Just wanted to let you know what I see on my end. Either way this line ended up getting modified

File: .github/workflows/build.yml (hunk 13 above)
Comment (itsmostafa) [reply .github/workflows/build.yml]: Sorry about that. I removed this earlier and auto formatter must have added it back. i'll remove it again shortly.

File: .github/workflows/build.yml (hunk 16 above)
Comment (itsmostafa) [reply .github/workflows/build.yml:33]: @dav3r resolved with [06bd355](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/06bd355907f2ed9b9f602b6b3ab27dbb1db394bf)

File: .github/workflows/build.yml (hunk 14 above)
Comment (itsmostafa) [reply .github/workflows/build.yml]: @jsf9k resolved with [06bd355](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/06bd355907f2ed9b9f602b6b3ab27dbb1db394bf)

File: .github/workflows/build.yml (hunk 13 above)
Comment (itsmostafa) [reply .github/workflows/build.yml]: @jsf9k resolved with [06bd355](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/06bd355907f2ed9b9f602b6b3ab27dbb1db394bf)

File: README.md (hunk 15 above)
Comment (itsmostafa) [reply README.md]: @jsf9k resolved with [8bd475c](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/8bd475c699b223b12b09dfa9503bc1cd7da6cc8c)

File: .github/workflows/build.yml (hunk 14 above)
Comment (jsf9k) [reply .github/workflows/build.yml]: A quick `git diff` will show you what changes have taken place.  I recommend _always_ doing that before a `git add`, just to make sure there is nothing unintentional.

Nota bene:  If you only want to add a subset of the changes that `git diff` shows for a file, you can do `git add -p` and select only the pieces you want to commit.

**Comment by jsf9k** [comment]
@itsmostafa - Once the tests are passing, and assuming I don't see any issues with the changes that are made make that happen, I will approve.

**Comment by itsmostafa** [comment]
@jsf9k tests pass now with @mcdonnnj suggestion to add gcompat lib dependancy in Dockerfile

**Comment by jsf9k** [comment]
> @jsf9k tests pass now with @mcdonnnj suggestion to add gcompat lib dependancy in Dockerfile

Can you or @mcdonnnj add a comment (either in this PR or in the `Dockerfile`) explaining why that is necessary and sufficient here?  I know that Alpine uses `musl` instead of `glibc`, and in the past I've treated this sort of failure as a good indicator that Alpine isn't a good fit.

**Comment by itsmostafa** [comment]
> > @jsf9k tests pass now with @mcdonnnj suggestion to add gcompat lib dependancy in Dockerfile
> 
> Can you or @mcdonnnj add a comment (either in this PR or in the `Dockerfile`) explaining why that is necessary and sufficient here? I know that Alpine uses `musl` instead of `glibc`, and in the past I've treated this sort of failure as a good indicator that Alpine isn't a good fit.

@jsf9k i actually dont need glibc. I just removed it with commit [3410827](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/34108275b859e444c1ec8df78705d785a2a6631a)

Per @mcdonnnj suggestion, i installed gcompat instead which is a C Library compatibility layer for `musl`

File: .github/workflows/build.yml (hunk 16 above)
Comment (dav3r) [reply .github/workflows/build.yml:33]: I was thinking of something more along the lines of:
```
# These are the only platforms currently supported by cisagov/con-pca-tasks
```

**Review by jsf9k — APPROVED** [review]

File: .github/workflows/build.yml (hunk 16 above)
Comment (itsmostafa) [reply .github/workflows/build.yml:33]: @dav3r resolved with [6b91a13](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/6b91a13f4cf005a7a3260e608f2d7b5695e3e59f)

File: tests/container_test.py [hunk 17]
```diff
@@ -8,9 +8,7 @@
 # Third-Party Libraries
 import pytest
 
-ENV_VAR = "ECHO_MESSAGE"
-ENV_VAR_VAL = "Hello World from docker compose!"
-READY_MESSAGE = "This is a debug message"
+READY_MESSAGE = "listening on port"
 SECRET_QUOTE = (
     "There are no secrets better kept than the secrets everybody guesses."  # nosec
 )
```
Comment (dav3r) [inline tests/container_test.py]: The `SECRET_QUOTE` should be removed, as it is now unused.

File: tests/container_test.py [hunk 18]
```diff
@@ -42,17 +40,9 @@ def test_wait_for_ready(main_container):
...
-
-
-def test_output(main_container):
-    """Verify the container had the correct output."""
-    main_container.wait()  # make sure container exited if running test isolated
-    log_output = main_container.logs().decode("utf-8")
-    assert SECRET_QUOTE in log_output, "Secret not found in log output."
+    main_container.stop()
+    assert main_container.wait() == 143, "Container service (main) exited"
+    assert version_container.wait() == 0, "Container service (version) exited"
```
Comment (dav3r) [inline tests/container_test.py]: The messages here are meant to state what happened if the assertion was not met - I believe you want messages like this, but please confirm:
```suggestion
    assert main_container.wait() == 143, "Container service (main) did not stop cleanly"
    assert version_container.wait() == 0, "Container service (version) did not exit cleanly"
```

**Review by dav3r — CHANGES_REQUESTED** [review]
I've got a couple more small test-related items and then I think I'm satisfied.

File: tests/container_test.py (hunk 17 above)
Comment (itsmostafa) [reply tests/container_test.py]: @dav3r forgot to remove this. resolved with [4b1d0a2](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/4b1d0a2beb4f09892a44d16eda557ace2b042b3f)

File: tests/container_test.py (hunk 18 above)
Comment (itsmostafa) [reply tests/container_test.py]: @dav3r that makes sense. resolved with [4b1d0a2](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/4b1d0a2beb4f09892a44d16eda557ace2b042b3f)

**Review by dav3r — APPROVED** [review]
This looks good to my eyes. 👍
